fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ImmutableAvailability.java

```diff
@@ -97,11 +101,15 @@ public ImmutableAvailability(
                 entry -> new SimplifiedIntervalList(entry.getValue());
 
         Map<Column, List<Interval>> map = dimensionIntervals.entrySet().stream()
+                .filter(entry -> dimensionDictionary.findByApiName(entry.getKey()) != null)
```

Comment:
I would put it at `trace` rather than debug, since it's expected to happen during a migration, but I agree, it might not be a bad thing to indicate that something is being filtered out. I really wish I understood grouped streams better, because "split on include/exclude, log the exclude" seems to be a very useful pattern.